grading/unitex-nlp/README.md

```diff
@@ -0,0 +1,23 @@
+INGInious
+=========
+
+This container is part of [INGInious](https://github.com/UCL-INGI/INGInious), an intelligent grader that allows secured and automated testing of code made by students. 
+
+Container with the Unitex/GramLab NLP engine (ingi/inginious-c-unitex-nlp)
+---------------------------------------------------------------------------------
+
+A container with [Unitex/GramLab](https://www.unitexgramlab.org) installed. The container includes:
+- the [Unitex/GramLab C++ Core](https://github.com/UnitexGramLab/unitex-core)
+- the [Unitex/GramLab Language Resources](https://github.com/UnitexGramLab/unitex-lingua)
+
+The container does NOT include the Unitex/GramLab IDE.
+
+By default, Unitex v3.3 will be installed. You can install a different version with `--build-arg UNITEX_VERSION=<version>`.
+
+To grade an assignment made with Unitex, you only need the `UnitexToolLogger` binary. This binary can be found in the `/home/unitex-core-${UNITEX_VERSION}/bin/` directory and can be used for:
```

Comment:
I strongly recommend to add a symbolic link to the executable into the `/usr/local/bin` folder. This way it will be automatically available from the Linux PATH and exercises that do not require a specific version of the tool will be easier to maintain.

    ln -s /home/unitex-core-${UNITEX_VERSION}/bin/UnitexToolLogger /usr/local/bin/UnitexToolLogger

Reply:
That's a good point. I'll add it, thanks!